packages/api/src/auth/decoders/index.ts

```diff
@@ -33,6 +33,21 @@ const typesToDecoders: Record<
   ethereum: ethereum,
   nhost: noop,
   custom: noop,
+  supertokens: noop,
+}
+
+export const isAuthHeaderUsedByType: Record<SupportedAuthTypes, boolean> = {
+  auth0: true,
+  azureActiveDirectory: true,
+  netlify: true,
+  goTrue: true,
+  magicLink: true,
+  firebase: true,
+  supabase: true,
+  ethereum: true,
+  nhost: true,
+  custom: true,
+  supertokens: false,
 }
```

Comment:
The above indicates if an auth provider uses auth bearer tokens (value will be `true`), or uses another way to send session tokens (like with cookies). This has the downstream affect that if the value here is `false`, then `getAuthenticationContext` will not call `parseAuthorizationHeader` and `decodeToken` functions